core/src/components/avatar/test/basic/avatar.e2e.ts

```diff
@@ -14,11 +14,15 @@ configs({ directions: ['ltr'] }).forEach(({ config, screenshot, title }) => {
       const avatarChip = page.locator('#avatar-chip');
       const avatarItemStart = page.locator('#avatar-item-start');
       const avatarItemEnd = page.locator('#avatar-item-end');
+      const avatarCharactersDisabled = page.locator('#avatar-characters-disabled');
+      const avatarDisabled = page.locator('#avatar-disabled');
```

Comment:
Can we move these tests to only take screenshots for the `ionic` theme and also use `setContent` in order to capture just the avatar. Here is an example: https://github.com/ionic-team/ionic-framework/blob/40932d7b2d9f6650ed621e0183b9c70b74debff8/core/src/components/badge/test/shape/badge.e2e.ts#L7-L24

Please make sure not to include the `ion-item` in the screenshots.